tests/manage/mcg/test_bucket_policy.py

```diff
@@ -264,6 +265,11 @@ def test_object_actions(self, mcg_obj, bucket_factory):
         get_policy = get_bucket_policy(mcg_obj, obc_obj.bucket_name)
         logger.info(f"Got bucket policy: {get_policy['Policy']}")
 
+        # Put object using OBC account(as an Admin)
+        if float(config.ENV_DATA['ocs_version']) >= 4.6:
+            logger.info(f'Putting an object on bucket: {obc_obj.bucket_name} using user: {obc_obj.obc_account}')
+            assert s3_put_object(obc_obj, obc_obj.bucket_name, object_key, data), "Failed: Put Object"
```

Comment:
Can you please cut the long lines, for example like:
```suggestion
            logger.info(
                f'Putting an object on bucket: {obc_obj.bucket_name} using user: {obc_obj.obc_account}'
            )
            assert s3_put_object(
                obc_obj, obc_obj.bucket_name, object_key, data
                ), "Failed: Put Object"
```

Reply:
Ack